recipes/pcl/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pcl" %}
+{% set version = "1.8.1" %}
+{% set sha256 = "5a102a2fbe2ba77c775bf92c4a5d2e3d8170be53a68c3a76cfc72434ff7b9783" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/pointcloudlibrary/{{ name }}/archive/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py!=36]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - flann
+    - eigen
+    - boost
```

Comment:
Don't you need these in the run requirements too?

Reply:
The existence tests wouldn't check that the library would link properly

Reply:
Looks good other than this!

Reply:
@scopatz Take a look now. Does this answer the mail? (Eigen is a header only library needed only at compile time, hence it's exclusion.)